### ISSUE TITLE
Quansight Labs

### ISSUE THREAD

**Issue by jbradberry** [python/planet#517]
## Problem
The articles in the Quansight Labs Blog feed are appearing to update every time they get pulled by Planet Python, causing these "Hello world" trial posts to wind up on the top of the page every day.

cc @rgommers

related #353 

## Details
![Screenshot 2022-07-12 at 10-06-15 Planet Python](https://user-images.githubusercontent.com/685957/178509641-777445f1-52bf-49bf-ade2-5d76438fddeb.png)

**Comment by rgommers** [comment]
Ouch, that's no good, sorry about that. Our website just got a complete overhaul. I don't immediately know where the problem is located, so how about we disable the feed for now and it's on me/us to open a new PR to re-enable it once we've validated that the feed works correctly? If that sounds good, do you want me to open a PR for that?

**Comment by morphex** [comment]
I was about to report this, but see that it's been 2 weeks already since someone did.

Did somebody leave a turd before taking a vacation?

**Comment by bskinn** [comment]
Our ***sincere*** apologies for our RSS feed causing this noise at the top of the Planet Python page for over three weeks. As @rgommers said, this is downstream of Quansight's website overhaul, and the issue got overlooked amid all of the other churn, organizing things after the site went live.

I've removed the offending RSS entries from our `rss.xml` -- once that change is merged through to live deployment, the issue should be fixed. Please ping me here if the problem persists.

**Comment by bskinn** [comment]
The fix has been merged to live.

**Comment by matrixise** [comment]
I think we can close this issue. Have a nice day.
